bbq/vm/test/utils.go

```diff
@@ -514,17 +518,9 @@ func CompileAndPrepareToInvoke(t testing.TB, code string, options CompilerAndVMO
 		programs,
 	)
 
-	// Ensure the program can be printed
-	const resolve = false
-	const colorize = false
-	printer := bbq.NewInstructionsProgramPrinter(resolve, colorize)
-	_ = printer.PrintProgram(program)
-
 	vmConfig := prepareVMConfig(t, options.VMConfig, programs)
 
-	if vmConfig.TypeLoader == nil {
-		vmConfig.TypeLoader = typeLoader(t, programs)
-	}
+	vmConfig.WithDebugEnabled()
```

Comment:
This will print the error and the instruction where the error occurred (only if an unhandled error occurred).